### PR TITLE
Release 1.9.0

### PR DESCRIPTION
**Pull request by WanzenBug**
* Updated default images:
  * LINSTOR 1.19.1
  * LINSTOR CSI 0.20.0
  * DRBD 9.1.8
  * DRBD-Reactor 0.8.0
* Disabled legacy HA Controller by default
* LINSTOR Affinity Controller referenced

### CODE REVIEW

File: charts/piraeus/values.yaml [hunk 1]
```diff
@@ -100,16 +100,16 @@ operator:
...
     sslSecret: ""
     automaticStorageType: None
     affinity: {}
     tolerations: []
     resources: {}
-    monitoringImage: quay.io/piraeusdatastore/drbd-reactor:v0.7.0
+    monitoringImage: quay.io/piraeusdatastore/drbd-reactor:v0.8.0
     monitoringBindAddress: ""
-    kernelModuleInjectionImage: quay.io/piraeusdatastore/drbd9-bionic:v9.1.7
+    kernelModuleInjectionImage: quay.io/piraeusdatastore/drbd9-bionic:v9.1.8
```
Comment (sribee) [inline charts/piraeus/values.yaml:112]: This comment is for the haController section - Does v1.9.0 continue to use old ha-controller or will it be changed to use new release of ha-controller?

Comment (WanzenBug) [reply charts/piraeus/values.yaml:112]: Done now, but still waiting on https://github.com/piraeusdatastore/helm-charts/pull/13 to be merged

**Comment by WanzenBug** [comment]
@rck sorry to spring this to you, but I think everyone else who can review this is out :/

It's mostly just updating some versions and a few docs, so it shouldn't be too hard.

**Review by rck — APPROVED** [review]
LGTM

**Pull request merged**